src/ui/VideoPreview/LargeVideoDetails.tsx

```diff
@@ -1,31 +1,31 @@
 import { ReactElement } from 'react'
 
 interface VideoDetailsProps {
-  title: string
+  largeVideoTitle: string
   author: string
   verified: boolean
   numberOfViews: string
   timePostedAgo: string
 }
 
-const VideoDetails = ({
-  title,
+const LargeVideoDetails = ({
```

Comment:
Also the component name(based on the folder) is `VideoPreview` why is this labeled as large video?